### PR TITLE
Happy number

### PR DESCRIPTION
**Pull request by salonishah01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Happy Number code in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1308

### CODE REVIEW

File: Python/other/HappyNumber.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Space Complexity: O(logN)
+
+Sample Input:
+Enter a number: 13
+Sample Output:
+13 is a Happy number
+
+
+
+
```
Comment (atarax665) [inline Python/other/HappyNumber.py:40]: Remove these blank lines.

**Review by atarax665 — CHANGES_REQUESTED** [review]
You have made too many changes in README. Revert it and just add the hyperlink of your contributed code. Everything else looks fine.

File: Python/other/HappyNumber.py (hunk 1 above)
Comment (salonishah01) [reply Python/other/HappyNumber.py:40]: @atarax665 Done

File: Python/other/HappyNumber.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ 
+"""
+Time Complexity: O(logN)
+Space Complexity: O(logN)
+
+Sample Input:
+Enter a number: 13
+Sample Output:
+13 is a Happy number
+
```
Comment (atarax665) [inline Python/other/HappyNumber.py:37]:
```suggestion
"""
```

**Review by atarax665 — COMMENTED** [review]
You have made so many unnecessary changes in README. Do revert them and just link your program under ```other``` heading.

**Comment by salonishah01** [comment]
@atarax665 It was getting difficult to resolve the changes committed. I have generated another PR #1373 , with correct updations.

**Pull request closed without merging**